fixtures/signer-config.json

```diff
@@ -0,0 +1,16 @@
+{
+	"server": {
+		"http_addr": ":4444",
+		"grpc_addr": ":7899",
+		"tls_cert_file": "./notary-signer.crt",
+		"tls_key_file": "./notary-signer.key",
+		"client_ca_file": "./notary-server.crt"
+	},
+	"logging": {
+		"level": "debug"
+	},
+	"storage": {
+		"backend": "mysql",
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
Possibly maybe enable sqlite3, so that if they don't run the container with `--rm` they can copy the sqlite file out, dump the DB, and migrate to a real DB?